### PR TITLE
Update create.md , ifx spell error, change statment to statement

### PR DESCRIPTION
**Pull request by yanfangli85**
## What is the purpose of the change

*(For example: This pull request makes task deployment go through the blob server, rather than through RPC. That way we avoid re-transferring them on each deployment (during recovery).)*

## Brief change log

*(for example:)*
  - *The TaskInfo is stored in the blob store on job creation time as a persistent artifact*
  - *Deployments RPC transmits only the blob storage reference*
  - *TaskManagers retrieve the TaskInfo from the blob cache*

## Verifying this change

Please make sure both new and modified tests in this PR follows the conventions defined in our code quality guide: https://flink.apache.org/contributing/code-style-and-quality-common.html#testing

*(Please pick either of the following options)*

This change is a trivial rework / code cleanup without any test coverage.

*(or)*

This change is already covered by existing tests, such as *(please describe tests)*.

*(or)*

This change added tests and can be verified as follows:

*(example:)*
  - *Added integration tests for end-to-end deployment with large payloads (100MB)*
  - *Extended integration test for recovery after master (JobManager) failure*
  - *Added test that validates that TaskInfo is transferred only once across recoveries*
  - *Manually verified the change by running a 4 node cluster with 2 JobManagers and 4 TaskManagers, a stateful streaming program, and killing one JobManager and two TaskManagers during the execution, verifying that recovery happens correctly.*

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (yes / no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: (yes / no)
  - The serializers: (yes / no / don't know)
  - The runtime per-record code paths (performance sensitive): (yes / no / don't know)
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Kubernetes/Yarn, ZooKeeper: (yes / no / don't know)
  - The S3 file system connector: (yes / no / don't know)

## Documentation

  - Does this pull request introduce a new feature? (yes / no)
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

File: docs/content.zh/docs/connectors/datastream/filesystem.md [hunk 1]
```diff
@@ -105,7 +105,7 @@ final FileSource<String> source =
 {{< tab "Python" >}}
 ```python
 source = FileSource.for_record_stream_format(...) \
-    .monitor_continously(Duration.of_millis(5)) \
```
Comment (davidradl) [inline docs/content.zh/docs/connectors/datastream/filesystem.md:108]: don't you need to do the english version too? Or is this done in another pr?

File: docs/content/docs/dev/table/hive-compatibility/hive-dialect/create.md [hunk 2]
```diff
@@ -73,7 +73,7 @@ CREATE [EXTERNAL] TABLE [IF NOT EXISTS] [db_name.]table_name
   ]
   [LOCATION fs_path]
   [TBLPROPERTIES (property_name=property_value, ...)]
-  [AS select_statment];
+  [AS select_statement];
```
Comment (davidradl) [inline docs/content/docs/dev/table/hive-compatibility/hive-dialect/create.md:76]: don't you need to do the chinese version too? Or is this done in another pr?

**Review by davidradl — COMMENTED** [review]
suggested changes - there seems to be missing files across the 2 languages

**Comment by XComp** [comment]
Thanks for your contribution, @yanfangli85 and for looking into this, @davidradl. David is right: We should do documentation changes in both the English and Chinese version at the same time to keep them on par. I'm gonna close this PR in favor of your first (as far as I can see?) PR #23422 to reduce the clutter. Feel free to put the changes in individual hotfix commits within PR #23422. That helps keeping the discussion in a single location. Creating individual PRs for each typo change seems to be a bit too much overhead.

Feel free to ask if you need help with creating additional commits in the existing PR (if that's the reason why you create multiple PRs).
